### DISCUSSION TITLE
MS-105 Dimmer problem

### DISCUSSION THREAD

**Discussion by kivanc87** [RavenSystem/esp-homekit-devices#1965] [MEPLHAA Scripts]
11.9 After the update, the device turns itself on and off.  Help please

{"c":{"r":[{"n":0,"s":9600}],"n":"HomeKit","b":[{"g":0,"t":5}]},"a":[{"t":30,"b":[{"g":0}],"0":{"u":[{"v":"55aa0006000501010001000d"}]},"1":{"u":[{"v":"55aa0006000501010001010e"}]},"y0":[{"v":1,"0":{"u":[{"v":"55aa00060008020200040000000a1f"}]}},{"v":5,"0":{"u":[{"v":"55aa00060008020200040000003247"}]}},{"v":10,"0":{"u":[{"v":"55aa00060008020200040000002e43"}]}},{"v":15,"0":{"u":[{"v":"55aa000600080202000400000098ad"}]}},{"v":20,"0":{"u":[{"v":"55aa0006000802020004000000c5da"}]}},{"v":25,"0":{"u":[{"v":"55aa0006000802020004000000fd12"}]}},{"v":30,"0":{"u":[{"v":"55aa0006000802020004000001253b"}]}},{"v":35,"0":{"u":[{"v":"55aa0006000802020004000001596f"}]}},{"v":40,"0":{"u":[{"v":"55aa00060008020200040000018ea4"}]}},{"v":45,"0":{"u":[{"v":"55aa0006000802020004000001c1d7"}]}},{"v":50,"0":{"u":[{"v":"55aa0006000802020004000001f208"}]}},{"v":55,"0":{"u":[{"v":"55aa0006000802020004000002263d"}]}},{"v":60,"0":{"u":[{"v":"55aa0006000802020004000002586f"}]}},{"v":65,"0":{"u":[{"v":"55aa000600080202000400000289a0"}]}},{"v":70,"0":{"u":[{"v":"55aa0006000802020004000002c0d7"}]}},{"v":75,"0":{"u":[{"v":"55aa0006000802020004000002ea01"}]}},{"v":80,"0":{"u":[{"v":"55aa00060008020200040000032038"}]}},{"v":85,"0":{"u":[{"v":"55aa00060008020200040000035169"}]}},{"v":90,"0":{"u":[{"v":"55aa00060008020200040000037f97"}]}},{"v":95,"0":{"u":[{"v":"55aa0006000802020004000003b7cf"}]}},{"v":100,"0":{"u":[{"v":"55aa0006000802020004000003e800"}]}}]}]}

**Comment by RavenSystem** [comment]
You must activate internal pull-up resistor with "p":1 when it is needed:

{"c":{"r":[{"n":0,"s":9600}],"n":"HomeKit","b":[{"g":0,"p":1,"t":5}]},"a":[{"t":30,"b":[{"g":0}],"0":{"u":[{"v":"55aa0006000501010001000d"}]},"1":{"u":[{"v":"55aa0006000501010001010e"}]},"y0":[{"v":1,"0":{"u":[{"v":"55aa00060008020200040000000a1f"}]}},{"v":5,"0":{"u":[{"v":"55aa00060008020200040000003247"}]}},{"v":10,"0":{"u":[{"v":"55aa00060008020200040000002e43"}]}},{"v":15,"0":{"u":[{"v":"55aa000600080202000400000098ad"}]}},{"v":20,"0":{"u":[{"v":"55aa0006000802020004000000c5da"}]}},{"v":25,"0":{"u":[{"v":"55aa0006000802020004000000fd12"}]}},{"v":30,"0":{"u":[{"v":"55aa0006000802020004000001253b"}]}},{"v":35,"0":{"u":[{"v":"55aa0006000802020004000001596f"}]}},{"v":40,"0":{"u":[{"v":"55aa00060008020200040000018ea4"}]}},{"v":45,"0":{"u":[{"v":"55aa0006000802020004000001c1d7"}]}},{"v":50,"0":{"u":[{"v":"55aa0006000802020004000001f208"}]}},{"v":55,"0":{"u":[{"v":"55aa0006000802020004000002263d"}]}},{"v":60,"0":{"u":[{"v":"55aa0006000802020004000002586f"}]}},{"v":65,"0":{"u":[{"v":"55aa000600080202000400000289a0"}]}},{"v":70,"0":{"u":[{"v":"55aa0006000802020004000002c0d7"}]}},{"v":75,"0":{"u":[{"v":"55aa0006000802020004000002ea01"}]}},{"v":80,"0":{"u":[{"v":"55aa00060008020200040000032038"}]}},{"v":85,"0":{"u":[{"v":"55aa00060008020200040000035169"}]}},{"v":90,"0":{"u":[{"v":"55aa00060008020200040000037f97"}]}},{"v":95,"0":{"u":[{"v":"55aa0006000802020004000003b7cf"}]}},{"v":100,"0":{"u":[{"v":"55aa0006000802020004000003e800"}]}}]}]}

**Reply by kivanc87** [reply]
Ok thanks 👍
